src/workerd/io/worker.c++

```diff
@@ -148,8 +148,12 @@ void addExceptionToTrace(jsg::Lock& js,
   }
 
   auto timestamp = ioContext.now();
-  auto error = KJ_REQUIRE_NONNULL(errorTypeHandler.tryUnwrap(js, exception),
-      "Should always be possible to unwrap error interface from an object.");
+  Worker::Api::ErrorInterface error;
+
+  if (exception.isObject()) {
```

Comment:
I guess the `isObject()` check is needed here because the deserialized result isn't necessarily an Object?  (But would typically be an Object?)

Reply:
Annoyingly, JavaScript permits you to throw any value, not just objects. I think there was a bug here that caused us to fail to log non-object exceptions, but it wasn't noticed before because our exception tunneling would turn non-objects into objects. (Exception tunneling today always produces `Error` values even if the original wasn't.)

Reply:
Ah, OK... that would explain why the current test cases for `throw "string"` were working without the Object check.